src/interface/variable.cpp

```diff
@@ -88,13 +88,13 @@ void CellVariable<T>::allocateComms(MeshBlock *pmb) {
   // set up fluxes
   std::string base_name = label();
   if (IsSet(Metadata::Independent)) {
-    flux[0] = ParArrayND<T>(base_name + ".flux0", GetDim(6), GetDim(5), GetDim(4),
+    flux[1] = ParArrayND<T>(base_name + ".flux1", GetDim(6), GetDim(5), GetDim(4),
                             GetDim(3), GetDim(2), GetDim(1));
     if (GetDim(2) > 1)
-      flux[1] = ParArrayND<T>(base_name + ".flux1", GetDim(6), GetDim(5), GetDim(4),
+      flux[2] = ParArrayND<T>(base_name + ".flux2", GetDim(6), GetDim(5), GetDim(4),
                               GetDim(3), GetDim(2), GetDim(1));
     if (GetDim(3) > 1)
-      flux[2] = ParArrayND<T>(base_name + ".flux2", GetDim(6), GetDim(5), GetDim(4),
+      flux[3] = ParArrayND<T>(base_name + ".flux3", GetDim(6), GetDim(5), GetDim(4),
```

Comment:
Perhaps name these flux**X**1, flux**X2**, and flux**X**3.

Reply:
thanks done.